datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -9488,6 +9501,18 @@ type AuditStamp {
     actor: String
 }
 
+type TimeStamp {
```

Comment:
At the GraphQL layer, we always use AuditStamp (simply because actor is an optional field here). We don't need a new timestamp type!

Reply:
fixed

Reply:
@jjoyce0510 does this mean we need to use audit stamp type in the mapper as well? (in that case we don't need timestamp mapper, we can just use pre-defined auditstamp)